nilearn/plotting/surf_plotting.py

```diff
@@ -422,6 +422,262 @@ def plot_surf_stat_map(surf_mesh, stat_map, bg_map=None,
     return display
 
 
+def _check_display_mode(display_mode):
+    """Checks whether the display_mode string passed to plot_img_on_surf
+    is meaningful.
+
+    display_mode: :obj:`str`
+        Any combination of 'lateral', 'medial', 'dorsal', 'ventral',
+        'anterior', 'posterior', united by a '+'. Ex.: 'dorsal+lateral'
+    """
+
+    available_modes = {'lateral', 'medial',
+                       'dorsal', 'ventral',
+                       'anterior', 'posterior'}
+
+    desired_modes = display_mode.split('+')
+
+    wrong_modes = set(desired_modes).difference(available_modes)
+    if wrong_modes:
+        msg = 'display_mode given: {}, available modes: {}'
+        msg = msg.format(wrong_modes, available_modes)
+        raise ValueError(msg)
+
+    return desired_modes
+
+
+def _check_hemisphere(hemisphere):
+    """Checks whether the hemisphere in plot_img_on_surf is correct.
+
+    hemisphere: :obj:`str`
+        Any combination of 'left' and 'right'
+    """
+    desired_hemispheres = hemisphere.split('+')
+    wrong_hemisphere = set(desired_hemispheres).difference({'left', 'right'})
+    if wrong_hemisphere:
+        raise ValueError('hemisphere only accepts left and right.')
+    if len(desired_hemispheres) > 2:
```

Comment:
Maybe it's my lack of imagination, but I don't see how we would ever accept 3 ? I find the error message somewhat confusing because of this.

Reply:
It's zombie code in any case. `wrong_hemisphere` can't be evaluated as `True` if there are more than two items.